peps/pep-0772.rst

```diff
@@ -387,10 +378,16 @@ serve.
 Elector eligibility
 -------------------
 
-Previous iterations of this PEP's draft proposed different membership rules for identifying the Packaging
-Council Electors.  After extensive discussion among stakeholders, and after seeking the widest possible
-feedback, the PEP authors agreed that aligning Packaging Council Electors with PSF Board voting membership was
-both the most equitable and most workable arrangement.
+Previous iterations of this PEP's draft proposed different membership rules for identifying the
+Packaging Council Electors.  After extensive discussion among stakeholders, and after seeking the
+widest possible feedback, the PEP authors agreed that aligning Packaging Council Electors with PSF
+Board voting membership was both the most workable arrangement and the most equitable approach to
+include all parts of the Python packaging community.
+
+PSF membership is used here because it is open to the broadest possible Python community. In
+particular, most people doing Python packaging work, do so in public, including contributing to both
+PyPA and non-PyPA projects, are likely to be eligible for PSF "Contributing Membership" based on
+that work, without paying any membership fee.
```

Comment:
small grammar fix, small wording fix

```suggestion
PSF membership is used here because it is open to the broadest possible Python community. In
particular, most people doing Python packaging work in public, including contributing to
PyPA or non-PyPA projects, are likely to be eligible for PSF "Contributing Membership" based on
that work, without paying any membership fee.
```

Reply:
Thanks.  I agree with the `s/and/or/` and the removal of "both".  The other grammar change was in a previous version and it didn't parse well for me so that's why I changed it.  But I get that my attempt was probably also inelegant.  Here's what I settled on:

> PSF membership is used here because it is open to the broadest possible Python community. In
particular, most people doing Python packaging work do such work in public, including contributing
to PyPA or non-PyPA projects, and are likely to be eligible for PSF "Contributing Membership" based
on that work without paying any membership fee.